pom.xml

```diff
@@ -13,6 +13,10 @@
   <packaging>hpi</packaging>
   <url>https://wiki.jenkins-ci.org/display/JENKINS/Compress+Build+Log+Plugin</url>
 
+  <properties>
+    <jenkins.version>2.7.4</jenkins.version>
```

Comment:
Lowest needed to make the plugin POM 2.30 restrictions work out, 1.651.x wasn't good enough.